firmware/tunerstudio/rusefi.input

```diff
@@ -7,15 +7,20 @@
 ; this should stop TS from looking for the CAN ID in the 2nd byte location and allow the page reads to work correctly.
 enable2ndByteCanID = false
 
-#unset tuneByMAF
+;#unset tuneByMAF
```

Comment:
did we intend to comment  #unset tuneByMAF? why?

Reply:
#unset instructed TS to create a project setting, and we don't need it anymore because now we define it explicitly in [SettingGroups] section (see below).